src/bidiMapper/scripts/eval.es

```diff
@@ -153,19 +159,11 @@
   }
 
   function evaluate(script, args) {
-    try {
       const deserializedArgs = args.map((arg) => deserialize(arg));
-      const func = new Function(`return (${script})`);
+      const func = new Function(`return (\n${script}\n)`);
```

Comment:
I was wondering why the added newlines here. @sadym-chromium says the answer is to preserve the column number. There's also an offset (in another PR) to adjust the numbers. In any case it's here for a reason, not accidental.